recipes/libvips/all/conanfile.py

```diff
@@ -0,0 +1,371 @@
+        "with_cfitsio": [True, False],
+        "with_cgif": [True, False],
+        "with_exif": [True, False],
+        "with_fftw": [True, False],
+        "with_fontconfig": [True, False],
+        "with_gsf": [True, False],
+        "with_heif": [True, False],
+        "with_imagequant": [True, False],
+        "with_jpeg": ["libjpeg", "libjpeg-turbo", "mozjpeg", False],
+        "with_jpeg_xl": [True, False],
+        "with_lcms": [True, False],
+        "with_magick": [True, False],
+        "with_matio": [True, False],
+        "with_nifti": [True, False],
+        "with_openexr": [True, False],
+        "with_openjpeg": [True, False],
+        "with_openslide": [True, False],
+        "with_orc": [True, False],
+        "with_pangocairo": [True, False],
+        "with_pdfium": [True, False],
+        "with_png": ["libpng", "libspng", False],
+        "with_poppler": [True, False],
+        "with_quantizr": [True, False],
+        "with_rsvg": [True, False],
+        "with_tiff": [True, False],
+        "with_webp": [True, False],
+        "with_zlib": [True, False],
+        "with_nsgif": [True, False],
+        "with_ppm": [True, False],
+        "with_analyse": [True, False],
+        "with_radiance": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "deprecated": True,
+        "cpp": True,
+        "introspection": False,
+        "vapi": False,
+        "with_cfitsio": False,
+        "with_cgif": False,
+        "with_exif": False,
+        "with_fftw": True,
+        "with_fontconfig": False,
+        "with_gsf": False,
+        "with_heif": False,
+        "with_imagequant": False,
+        "with_jpeg": "libjpeg",
+        "with_jpeg_xl": False,
+        "with_lcms": True,
+        "with_magick": False,
+        "with_matio": False,
+        "with_nifti": False,
+        "with_openexr": False,
+        "with_openjpeg": False,
+        "with_openslide": False,
+        "with_orc": False,
+        "with_pangocairo": False,
+        "with_pdfium": False,
+        "with_png": "libspng",
+        "with_poppler": False,
+        "with_quantizr": False,
+        "with_rsvg": False,
+        "with_tiff": True,
+        "with_webp": True,
+        "with_zlib": True,
+        "with_nsgif": True,
+        "with_ppm": True,
+        "with_analyse": True,
+        "with_radiance": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        if not self.options.cpp:
+            self.settings.rm_safe("compiler.cppstd")
+            self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("expat/2.5.0")
+        self.requires("glib/2.76.0", transitive_headers=True, transitive_libs=True, run=can_run(self))
```

Comment:
😄 Please comment why the libs are transitive 🙏

Reply:
glib headers are included in most public headers of libvips, and several glib functions are called in these headers, so not sure what kind of comment you want that would be more useful than a self-explanatory `transitive_headers=True, transitive_libs=True`.